src/functionalTest/java/uk/gov/hmcts/reform/draftstore/endpoint/v3/GetAllTest.java

```diff
@@ -31,11 +32,15 @@
     private MockMvc mockMvc;
 
     @MockBean private DraftService draftService;
-    @MockBean private AuthService authService; //NOPMD - mock declaration required
```

Comment:
Does PMD no longer complain about it?

Reply:
AuthService is not used below, so the comment is not longer needed

Reply:
👍